Source/Resource/Plugin/Language/fr.csv

```diff
@@ -934,293 +934,293 @@ Language_wo;Wolof
+RenderMode;Type de rendu
 Renew;Renouveler
 ReplayGain_Gain;Gain en lecture
 ReplayGain_Peak;Crête de gain en lecture
 Report;Rapport
 Resolution;Résolution
 RestoreLifetimeSubscription;Restaurer l'abonnement à vie
-s;s
-SamplePeakLevel;
-SamplePeakLevel_Album;
-SamplesPerFrame;
+s; s
+SamplePeakLevel;Niveau crête de l'échantillon
+SamplePeakLevel_Album;Niveau crête de l'échantillon (album)
+SamplesPerFrame;Échantillons par image
 SamplingCount;Nombre d'échantillons
-SamplingRate;Echantillonnage
+SamplingRate;Échantillonnage
 Save;Sauvegarder
 ScanOrder;Ordre de balayage
-ScanOrder_Original;Ordre original de balayage
-ScanOrder_Stored;Ordre stocké de balayage
-ScanOrder_StoredDisplayedInverted;Ordre de balayage invversé entre affiché/stocké
-ScanOrder_StoreMethod;Ordre de balayage, mode d'enregistrement
+ScanOrder_Original;Ordre original
+ScanOrder_Stored;Ordre stocké
+ScanOrder_StoredDisplayedInverted;Ordre inversé entre Affiché/Stocké
+ScanOrder_StoreMethod;Mode d'enregistrement de l'ordre
 ScanType;Type de balayage
-ScanType_Original;Type de balayage original
-ScanType_StoreMethod;Ordre de balayage, mode d'enregistrement
+ScanType_Original;Original
+ScanType_StoreMethod;Méthode d'enregistrement
 ScreenplayBy;Joué par
-ScreenToCenter;
-ScreenToFront;
+ScreenToCenter;Écran au centre
+ScreenToFront;Écran à l'avant
 Season;Saison
-SecondaryAudioLanguage;
-see below;voir ci-dessous
-Send HeaderFile;SVP envoyez-moi le fichier Header ici : http://sourceforge.net/projects/mediainfo/ (partie Bugs)
-Separator_Columns;Séparateur de colonnes
-Separator_Lines;Séparateur de lignes
-SeriesTitle;
-ServiceChannel;Numéro du service
-ServiceKind;
+SecondaryAudioLanguage;Langue audio secondaire
+see below;Voir ci-dessous
+Send HeaderFile;Merci de m'envoyer le fichier Header ici : http://sourceforge.net/projects/mediainfo/ (partie Bugs)
+Separator_Columns;Séparateur de colonnes :
+Separator_Lines;Séparateur de lignes :
+SeriesTitle;Titre de la série
+ServiceChannel;Canal du service
+ServiceKind;Genre de service
 ServiceName;Nom du service
 ServiceProvider;Fournisseur du service
 ServiceType;Type de service
-Set;Ensemble
-Set_Count;Nombre d'ensembles
+Set;Réglage
+Set_Count;Nombre de réglages
 Setup;Configuration
 Sharpness;Netteté
 Sheet;Tableau
 Sheet (Complete);Tableau (Complet)
-Shell extension;Extension Shell (clic droit sur le fichier, il y aura un menu 'Media Info')
-Shell extension, folder;Pour les répertoires aussi
-Shell InfoTip;Extension bulle d'aide (passez la souris sur un fichier, il y aura toutes les infos)
-ShimName;
-ShimVersion;
-Show menu;Montrer le menu
-Show toolbar;Montrer la barre d'outils
-SignalGroup;
-SigningPresent;
-SignLanguage;
+Shell extension;Extension du Shell (le clic droit sur un fichier multimédia propose un choix 'Media Info')
+Shell extension, folder;Extension du Shell pour les répertoires
+Shell InfoTip;Bulle d'aide 'Explorateur' (survoller un fichier multimedia à la souris affichera toutes ses infos)
+ShimName;Nom du calage

```

Comment:
From https://www.wordreference.com/enfr/shim , it seems that for computer science there is no translation, so keep "shim".